crates/volta-migrate/src/v3/config.rs

```diff
@@ -18,6 +18,8 @@ pub struct LegacyPackageConfig {
 pub struct LegacyPlatform {
     pub node: NodeVersion,
     #[serde(with = "option_version_serde")]
+    pub pnpm: Option<Version>,
+    #[serde(with = "option_version_serde")]
```

Comment:
I think we can actually omit this, since the `LegacyPlatform` necessarily won't have a `pnpm` value, as we didn't support it until this point. So we can default the value to `None` for those packages that are being migrated.